dashboards/aerospike/README.md

```diff
@@ -0,0 +1,21 @@
+# Alerts for Aerospike in the Ops Agent
```

Comment:
It looks like this readme describes the alerts. Can you update this to reflect the dashboard content, and match the format of the other READMEs, e.g. https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/blob/master/dashboards/apache/README.md

Reply:
Ah sorry about that, I was making the Alerts PR at the same time. Just fixed this